packages/core/contracts/insured-bridge/ovm/OVM_CrossDomainEnabled.sol

```diff
@@ -1,84 +0,0 @@
-// Copied directly from https://github.com/ethereum-optimism/optimism/blob/7ffb83509c589bd35c2e5f9cb2c6ccbd5f346175/packages/contracts/contracts/optimistic-ethereum/libraries/bridge/OVM_CrossDomainEnabled.sol
```

Comment:
This was redundant with the OVM_CrossDomainEnabled contract in `external`, so I remove this one in favor of the `external` one.